tests/nat/agent/test_rewoo.py

```
@@ -861,3 +881,136 @@ async def test_executor_node_raise_tool_call_error_success_case(mock_llm, mock_t
+
+def test_dependency_parsing_complex():
+    """Test dependency parsing for complex dependency graph."""
+    steps = [
+        _create_step_info("step1", "#E1", "tool_A", "input1"),
+        _create_step_info("step2", "#E2", "tool_B", "input2"),
+        _create_step_info("step3", "#E3", "tool_C", "#E1"),
+        _create_step_info("step4", "#E4", "tool_D", "#E2"),
+        _create_step_info("step5", "#E5", "tool_E", {"inputs": ["#E3", "#E4"]})
+    ]
+
+    evidence_map, execution_levels = ReWOOAgentGraph._parse_planner_dependencies(steps)
```

Ruff (0.13.1)

Unpacked variable `evidence_map` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)